python/data_extraction/content_export.py

```diff
@@ -22,6 +28,20 @@ def content_hash_slicer(content_hash, base_fields=[], taxon_fields=[], ppo_field
     return result
 
 
+def untagged_hash_slicer(content_hash, base_fields=[], ppo_fields=[]):
```

Comment:
Maybe say `untagged_content_slicer`, or `untagged_dict_slicer` as `hash` is not a thing in Python.

Reply:
renamed hash->dict